MantidPlot/src/Mantid/FitParameterTie.cpp

```diff
@@ -10,9 +10,10 @@
 #include <stdexcept>
 
 /// Constructor
+// cppcheck-suppress passedByValue
 FitParameterTie::FitParameterTie(
     std::shared_ptr<Mantid::API::CompositeFunction> cf)
-    : m_compositeFunction(cf), m_prop(nullptr) {}
+    : m_compositeFunction(std::move(cf)), m_prop(nullptr) {}
```

Comment:
@martyngigg these suppressions do not seem to be having the desired effect... am I doing something wrong?